RecoCTPPS/TotemRPLocal/python/totemTimingLocalReconstruction_cff.py

```diff
@@ -2,12 +2,11 @@
 
 # reco hit production
 from RecoCTPPS.TotemRPLocal.totemTimingRecHits_cfi import totemTimingRecHits
-totemTimingRecHits.saturationLimit = cms.double(0.85)
-totemTimingRecHits.cfdFraction = cms.double(0.3)
-totemTimingRecHits.smoothingPoints = cms.int32(20)
-totemTimingRecHits.lowPassFrequency = cms.double(0.7)
-totemTimingRecHits.hysteresis = cms.double(5e-3)
+
+# local track fitting
+from RecoCTPPS.TotemRPLocal.totemTimingLocalTracks_cfi import totemTimingLocalTracks
 
 totemTimingLocalReconstruction = cms.Sequence(
-    totemTimingRecHits
+    totemTimingRecHits *
+    totemTimingLocalTracks
```

Comment:
I'm failing to find a ```keep``` statement for this.
I noticed only post-factum, after running locally.
Please add appropriate changes to RecoCTPPS/Configuration/python/RecoCTPPS_EventContent_cff.py

For the future (and perhaps other CTPPS PRs) please make sure that you test the code with the standard workflow, to make sure that everything is running and that the outputs are produced as expected.